package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "ng": "ng",
     "build": "npm run env -s && ng build --prod --output-hashing=none",
-    "build:prod": "node --max-old-space-size=16384 ./node_modules/@angular/cli/bin/ng build --prod --output-hashing=none --base-href=\"/web-app/\"",
+    "build:prod": "node --max-old-space-size=16384 ./node_modules/@angular/cli/bin/ng build --prod --output-hashing=none --base-href=/web-app/",
```

Comment:
Any specific reason for this ?

Reply:
I tested my ci with the syntax I wrote, that's why just changed it to follow my syntax. Might have worked with that one also, just tried to play safe.

Reply:
Alright.
The PR builds and deploys on your test repo are working fine, update the typos and we'll merge.

Reply:
Thanks would do this shortly. :)

Reply:
Will this syntax work if we execute this locally without using the CI?

Reply:
Yes, in any case, it would work. The below syntax is copied from the angular docs itself.
ng build --prod --output-path docs --base-href /<project_name>/
Source link: https://angular.io/guide/deployment#deploy-to-github-pages
Useful Links: https://angular.io/guide/deployment#the-base-tag
@abhaychawla